packages/postcss-reduce-initial/data/toInitial.json

```diff
@@ -17,17 +17,19 @@
   "box-sizing": "content-box",
   "column-rule-color": "currentcolor",
   "font-synthesis": "weight style",
+  "image-orientation": "from-image",
   "mask-clip": "border-box",
   "mask-mode": "match-source",
   "mask-origin": "border-box",
-  "mask-repeat": "repeat",
```

Comment:
The [spec](https://drafts.fxtf.org/css-masking/#the-mask-repeat) and [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/mask-repeat) do not agree on this one. Could be browser-dependent? A familiar person raised [an issue](https://github.com/mdn/data/issues/351) but the spec changed in between...

Reply:
Value is `repeat` and it goes into `fromInitial`.